### PR TITLE
[SYCL] Fix Plugin/level_zero_imm_cmdlist_per_thread.cpp test

### PR DESCRIPTION
**Pull request by aelovikov-intel**
Make it pass on devices that have both compute and copy engines.

### CODE REVIEW

**Comment by bader** [comment]
> Make it pass on devices that have both compute and copy engines.

Does it fix "flaky failure on windows"? See lines 3-4:

// Flaky failure on windows
// UNSUPPORTED: windows

**Comment by aelovikov-intel** [comment]
> Does it fix "flaky failure on windows"?

I don't know, but I wouldn't expect so.

**Review by rdeodhar — APPROVED** [review]
Yes, looks good.

File: sycl/test-e2e/Plugin/level_zero_imm_cmdlist_per_thread.cpp [hunk 1]
```diff
@@ -4,8 +4,8 @@
 // UNSUPPORTED: windows
 
 // RUN: %{build} %level_zero_options %threads_lib -o %t.out
-// RUN: env SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS=1 ZE_DEBUG=4 %{run} %t.out 2>&1 | FileCheck --check-prefixes=CHECK-ONE-CMDLIST %s
-// RUN: env SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS=2 ZE_DEBUG=4 %{run} %t.out 2>&1 | FileCheck --check-prefixes=CHECK-PER-THREAD-CMDLIST %s
+// RUN: env SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS=1 SYCL_PI_LEVEL_ZERO_USE_COPY_ENGINE=0 ZE_DEBUG=-1 %{run} %t.out 2>&1 | FileCheck --check-prefixes=CHECK-ONE-CMDLIST %s
+// RUN: env SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS=2 SYCL_PI_LEVEL_ZERO_USE_COPY_ENGINE=0 ZE_DEBUG=-1 %{run} %t.out 2>&1 | FileCheck --check-prefixes=CHECK-PER-THREAD-CMDLIST %s
```
Comment (aelovikov-intel) [inline sycl/test-e2e/Plugin/level_zero_imm_cmdlist_per_thread.cpp:8]:
```suggestion
// RUN: env SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS=1 SYCL_PI_LEVEL_ZERO_USE_COPY_ENGINE=0 ZE_DEBUG=4 %{run} %t.out 2>&1 | FileCheck --check-prefixes=CHECK-ONE-CMDLIST %s
// RUN: env SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS=2 SYCL_PI_LEVEL_ZERO_USE_COPY_ENGINE=0 ZE_DEBUG=4 %{run} %t.out 2>&1 | FileCheck --check-prefixes=CHECK-PER-THREAD-CMDLIST %s
```

**Comment by aelovikov-intel** [comment]
> Is there any reason to use ZE_DEBUG=-1 instead of the existing =4?

Remnants of local debug. Reverted that part.

**Review by smaslov-intel — APPROVED** [review]

**Review by againull — APPROVED** [review]

**Pull request merged**